src/main/java/redis/clients/jedis/commands/MultiKeyCommands.java

```diff
@@ -190,4 +192,10 @@
    * @return
    */
   List<Map.Entry<String, List<StreamEntry>>> xreadGroup(String groupname, String consumer, int count, long block, final boolean noAck, Map.Entry<String, StreamEntryID>... streams);
+
+  Long georadiusStore(String key, double longitude, double latitude, double radius,
+      GeoUnit unit, GeoRadiusParam param);
```

Comment:
It may be better to enforce user to provide the *store key* in GEORADIUS with STORE option.

One of the ways to do so could be altering this above method as:

```
Long georadiusStore(String key, double longitude, double latitude, double radius,
    GeoUnit unit, GeoRadiusParam param, String storeKey);
```

and keep GeoRadiusParam as it was.

Reply:
I think OK, for `store` and `storedist`, we need to add the following four commands:

```
  Long georadiusStore(String key, double longitude, double latitude, double radius,
      GeoUnit unit, GeoRadiusParam param, String storeKey);

  Long georadiusStoreDist(String key, double longitude, double latitude, double radius,
      GeoUnit unit, GeoRadiusParam param, String storeDistKey);

  Long georadiusByMemberStore(String key, String member, double radius, GeoUnit unit,
      GeoRadiusParam param, String storeKey);

  Long georadiusByMemberStoreDist(String key, String member, double radius, GeoUnit unit,
      GeoRadiusParam param, String storeDistKey);

```
WDYT?

Reply:
Another idea:

    Long georadiusStore(String key, double longitude, double latitude, double radius,
        GeoUnit unit, GeoRadiusParam param, GeoRadiusStoreParam storeParam);

    Long georadiusByMemberStore(String key, String member, double radius, GeoUnit unit,
        GeoRadiusParam param, GeoRadiusStoreParam storeParam);

@gkorland PING!

Reply:
@gkorland Please give your suggestions.

Reply:
@sazzad16 @gkorland PING

Reply:
@yangbodong22011 If you're really eager, you can implement alternate idea in separate branch/PR so that I'll be able to review it.

Reply:
@sazzad16 OK，I will do it.